### DISCUSSION TITLE
RfTimeOut support for Sonoff RF Bridge

### DISCUSSION THREAD

**Discussion by clau-bucur** [arendst/Tasmota#20937] [Ideas]
A while ago I opened a request on https://github.com/arendst/Tasmota/issues/10248 so that xdrv_06_snfbridge / SFB_TIME_AVOID_DUPLICATE is made configurable.
I see now that in the meanwhile, a new setting `RfTimeOut` has been added by https://github.com/arendst/Tasmota/commit/ae484e28ba6e71759731a8a09ce18b5541520c63

Would it be possible that the same `RfTimeOut` setting be used instead of the 2 seconds hardcoded value of SFB_TIME_AVOID_DUPLICATE?

**Comment by Jason2866** [comment]
@arendst The change would be small and should do no harm.

**Reply by arendst** [reply]
Will do.

**Reply by arendst** [reply]
Added to latest dev branch.
